.github/workflows/release.yml

```diff
@@ -544,7 +544,7 @@ jobs:
         id: download-binaries
         uses: actions/download-artifact@v4
         with:
-          pattern: artifacts-binary-${{env.NETWORK}}-${{github.run_id}}
+          pattern: artifacts-binary-local-${{github.run_id}}
```

Comment:
This was the main fix, but also swapped from dev to local so that we could also tag any parachain related pallet changes.